File: P4.c
// Accept Character from user and check whether it is small or not (a=z).

#include<stdio.h>
#include<stdbool.h>

bool CheckSmall(char *ptr)
{
    bool bflag = false;

    while (*ptr != '\0')
    {
        if((*ptr >= 'a') && (*ptr <= 'z'))
        {
            bflag = true;
        }
        else
        {
            bflag = false;
        }
        ptr++;
    }

    return bflag;
}
int main()
{
    char Arr[50] = {'\0'};
    bool iRet = false;

    printf("Enter the String : \n");
    scanf("%[^ \n]s",Arr);

    iRet = CheckSmall(Arr);

    if(iRet == true)
    {
        printf("True");
    }
    else
    {
        printf("False");
    }

    return 0;
}

File: Assignment 25/P4.java
/* Write a program which accept string from user and check whether
it contains vowels in it or not.
Input : "marvellous"
Output : TRUE
Input : "Demo"
Output: TRUE
Input : "xyz"
Output : FALSE */

import java.util.Scanner;

class P4
{
    public static boolean CheckVowel(String str)
    {
        boolean bFlag = false;

        char[] Arr = str.toCharArray();

        for(int i = 0;i < Arr.length;i++)
        {
            if(Arr[i] == 'a' || Arr[i] == 'e' || Arr[i] == 'i' || Arr[i] == 'o' || Arr[i] == 'u' ||
                Arr[i] == 'A' || Arr[i] == 'E' || Arr[i] == 'I' || Arr[i] == 'O' || Arr[i] == 'U' )
                {
                    bFlag = true;
                }
                else{
                    return bFlag;
                }       
        }
        return bFlag;
    }
    
    public static void main(String arg[])
    {
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Input : ");

        String str =  sobj.nextLine();

        bRet = CheckVowel(str);

        if(bRet == true)
        {
            System.out.println("TRUE");
        }
        else{
            System.out.println("False");

        }

        sobj.close();

    }
}





